crates/orchestrator/src/generators/bootnode_addr.rs

```diff
@@ -1,13 +1,12 @@
-use std::{fmt::Display, net::IpAddr};
+use std::fmt::Display;
 
 use super::errors::GeneratorError;
 
 pub fn generate<T: AsRef<str> + Display>(
     peer_id: &str,
-    ip: &IpAddr,
+    ip: &str,
```

Comment:
Why do we move from IpAddr to str ? We lose the type safety of ensuring the IpAddr is valid, wdyt ?

Reply:
Sounds good, let me rollback the change and add a parsing logic to get an `IpAddr` from String.